api/src/main/resources/api/service_openapi.yaml

```diff
@@ -876,3 +941,43 @@ components:
         - id
         - title
       additionalProperties: true
+
+    #########################UNDERLAY CONFIG V2 [START]#################################
+    UnderlayV2:
+      type: object
+      description: Underlay
+      properties:
+        name:
+          description: Name
+          type: string
+        description:
```

Comment:
Is this intended as a display name (e.g. `CMS SynPUF`) rather than a description (e.g. `Medicare Claims Synthetic Public Use Files (SynPUFs) were created to allow interested parties to gain familiarity using Medicare claims data while protecting beneficiary privacy.`)?

Reply:
I meant display name, yes. I'll update the parameter name to `displayName`.